generative/maisi/maisi_inference_tutorial.ipynb

```diff
@@ -0,0 +1,288 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    print("Environment variables have been read in.")
</pre><p><code>read in</code> -&gt; <code>loaded</code></p><p><code>Environment variables</code> -&gt; <code>Global config variables</code> (because environment variables are things like <code>CUDA_VISIBLE_DEVICES</code>)</p><p> </p><p>Consider adding this to visualize the inputs after the print</p>

<pre class="ql-syntax" spellcheck="false">for k, v in vars(args).items():
    print(f"{k}: {v}")
</pre><p>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1741/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Thanks, done